db/migrate/20171023170841_ensure_cloud_managers.rb

```diff
@@ -0,0 +1,47 @@
+class EnsureCloudManagers < ActiveRecord::Migration[5.0]
+  class ExtManagementSystem < ActiveRecord::Base; end
+  class BaseManager < ExtManagementSystem; end
+  class Settings < ActiveRecord::Base; end
```

Comment:
I don;t know why Settings is needed...Settings is not even an ActiveRecord model in the manageiq app.

Reply:
Settings (originally referenced as `::Settings`) is referred to within the `ensure_managers` method. I've modified it so that it's not a subclass of AR any more.